#1 create serverless notarization api

skolumban-ex opened the pull request.

Commit:
#1 Create serverless notarization API

=> first commit

Commit:
#1_Create serverless notarization API

=> migration

Commit:
#1_Create serverless notarization API

=> HashObject and ReHashObject endpoints

Commit:
#1_Create serverless notarization API

=> code cleanup

Commit:
Publish on push to brach

Commit:
Create release automation on master branch merge and remove unnecessary reference

Commit:
Update .net version of workflow images.

weigandf commented on ObjectHashServerr.API/Function1.cs:
```diff
@@ -0,0 +1,109 @@
+using ObjectHashServer.BLL.Services.Implementations;
+using ObjectHashServer.Controllers;
+
+namespace ObjectHashServerr.API
```
The name has a typo: ObjectHashServer with one r at the end - I guess ;)

weigandf commented on ObjectHashServerr.API/Function1.cs:
```diff
@@ -0,0 +1,109 @@
+
+namespace ObjectHashServerr.API
+{
+    public class Function1
```
Guess Function1 will be replaced with a meaningfull name

skolumban-ex replied:
@tbordi-ex  rename the class and file to ObjectHasherFunctions

weigandf reviewed (changes requested):
Small typo ObjectHashServerr - rest looks good. Thanks

skolumban-ex commented:
@tbordi-ex, once you fix the comments from @weigandf , you can merge into the master and delete this branch. Then merge the master into the branch of #5 , so the diff will be clean there.

Reassing that review to Florian, when the diff is only with respect to the most recent master.

weigandf commented:
@skolumban-ex @tbordi-ex any update?

skolumban-ex commented:
@weigandf , @tbordi-ex will work on the project again tomorrow, it will be actualized then.

Commit:
#1 create serverless notarization api #4

=> corrections after review + disabling nullables in ObjectHashServer.BLL.csproj
